build.gradle

```diff
@@ -24,16 +30,79 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
     antlr 'org.antlr:antlr4:4.5'
 
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    // using groovy 2.3 to avoid problems in eclipse
+    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
```

Comment:
I think that those people haver never used any other IDE… :smirk_cat: Eclipse is the worst IDE I’ve ever seen, it’s so extremely buggy that it’s hard to distinguish what’s a bad feature and what’s a bug.

Anyway, the point is that we should IMHO never take any decision just because of IDE xyz doesn’t work properly…